### ISSUE TITLE
The Matomo documentation about campaign tracking is misleading, campaign name is not required

### ISSUE THREAD

**Issue by Chardonneaur** [piwik/piwik#22233]
Hi team,
in the matomo documentation, it is written that the campaign name is necessary to trigger a campaign visit:
![Screenshot from 2024-05-17 16-41-55](https://github.com/matomo-org/matomo/assets/28002238/4673e0fc-1428-4493-a728-50d829706ea9)
whereas this is not true, in the screenshot below i just created a visit with only the parameter medium:
![Screenshot from 2024-05-17 16-42-43](https://github.com/matomo-org/matomo/assets/28002238/0cbdebd6-fb16-4a5e-b84a-ad4d2766438c)
Suggested solution:
to double check which parameter is necessary and to update the documentation accordingly.

**Comment by jmcourtiade** [comment]
Yes and btw the name 'mediä' is not proper in the French translation. This word does not exist like this ('media' is OK) but we could still use 'medium'.

**Comment by michalkleiner** [comment]
The translations can be updated via Weblate (https://hosted.weblate.org/projects/matomo/).

**Comment by jmcourtiade** [comment]
> The translations can be updated via Weblate (https://hosted.weblate.org/projects/matomo/).

Thank you, I have just registered to it and will update it.

**Comment by sgiehl** [comment]
@Chardonneaur The generator / documentation should be for core only. The campaign medium and stuff like that is detected by the MarketingCampaignsReporting plugin, which behaves and detects campaigns differently.

refs #19951, #20067

**Comment by Chardonneaur** [comment]
@sgiehl another user got confused about it today.
Here is the situation:
- when they look at channel type they see a number of visits for campaigns.
- when they at campaign names they are seeing a lower number of visits.
Is there any explanations why the campaign name is not required anymore?

**Comment by sgiehl** [comment]
Campaigns might not need to be tracked with names. And when using the MarketingCampaigns there might be additional reasons for different numbers if the plugin and core are not configured to use exatly the same parameters

**Comment by Chardonneaur** [comment]
@sgiehl ok, but in this case, why is it written that the campaign name is required?
Cannot we just change our documentation? https://matomo.org/faq/tracking-campaigns-url-builder/
